google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/RoutineInfo.java

```diff
@@ -506,6 +534,9 @@ Routine toPb() {
     if (getRemoteFunctionOptions() != null) {
       routinePb.setRemoteFunctionOptions(getRemoteFunctionOptions().toPb());
     }
+    if (getImportedLibraries() != null) {
```

Comment:
Was the lack of this conversion causing test failures?  If not, is there anything still missing from our roundtrip testing of routine metadata?